src/components/AmountPicker/index.tsx

```diff
@@ -29,7 +29,7 @@ function AmountPicker({value, description, title, errorText = '', onInputChange,
         hidePickerModal();
     };
 
-    const descStyle = !value || value.length === 0 ? StyleUtils.getFontSizeStyle(variables.fontSizeLabel) : null;
+    const descStyle = !value || value.length === 0 ? StyleUtils.getFontSizeStyle(variables.fontSizeNormal) : null;
```

Comment:
`MenuItemWithTopDescription` already handles this logic, so we actually don't need any of this and we can remove this variable and its usage from this component.

Reply:
Works for me!

Reply:
And just for my own reference, it looks like this is the logic that handles that yeah? From `MenuItem.tsx`

![CleanShot 2024-06-12 at 20 46 15@2x](https://github.com/Expensify/App/assets/2319350/3d377037-4729-47d4-8c42-e352f568c26e)

Reply:
correct